tests/edgeless/clipboard.spec.ts

```diff
@@ -228,3 +238,96 @@ test.describe('pasting URLs', () => {
     await expect(page.locator('bookmark-card')).toBeVisible();
   });
 });
+
+test.describe('pasting blocks', () => {
```

Comment:
we'd better put it in another file to speed up the ci.

Reply:
It is moved to `paste-block.spec.ts`

Reply:
Doone